include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -355,31 +372,42 @@ class MMR920 {
                         .message_index = m.message_index,
                         .severity = can::ids::ErrorSeverity::unrecoverable,
                         .error_code = can::ids::ErrorCode::over_pressure});
-            } else {
+            } else if (!bind_sync) {
+                // if we're not using bind sync turn off the sync line
+                // we don't do this during bind sync because if it's triggering
+                // the sync line on purpose this causes bouncing on the line
+                // that turns off the sync and then immediately turns it back on
+                // and this can cause disrupt the behavior
                 hardware.reset_sync();
             }
         }
         if (bind_sync) {
             if (std::fabs(pressure - current_pressure_baseline_pa) >
                 threshold_pascals) {
                 hardware.set_sync();
+                // force this to stay set long enough to debounce on other nodes
             } else {
                 hardware.reset_sync();
             }
         }
 
         if (echo_this_time) {
             auto response_pressure = pressure - current_pressure_baseline_pa;
-            // do we want pressure or response pressure
-            sensor_buffer_log(pressure);
-            can_client.send_can_message(
-                can::ids::NodeId::host,
-                can::messages::ReadFromSensorResponse{
-                    .message_index = m.message_index,
-                    .sensor = can::ids::SensorType::pressure,
-                    .sensor_id = sensor_id,
-                    .sensor_data =
-                        mmr920::reading_to_fixed_point(response_pressure)});
+
+            sensor_buffer_log(response_pressure);
+
+            if (sensor_buffer_index == 10 && !crossed_buffer_index) {
+
+                current_pressure_baseline_pa =
```

Comment:
why are we incrementing instead of setting this? won't this make it run away? also we could put `10*current_pressure_baseline_pa` in the init argument to `accumulate` if we do want the accumulation

Reply:
I'll add a comment that explains why this won't run away

Reply:
re-worked it to make it more clear